lib/core/src/Cardano/Wallet.hs

```diff
@@ -1887,7 +1889,7 @@ migrationPlanToSelectionWithdrawals plan rewardWithdrawal outputAddressesToCycle
 
         outputAddressesRemaining :: [Address]
         outputAddressesRemaining =
-            drop (length $ view #outputs migrationSelection) outputAddresses
+            drop (length outputsCovered) outputAddresses
```

Comment:
Is this a fix, or is it equivalent?

Reply:
> Is this a fix, or is it equivalent?

It's equivalent.

I guess, strictly speaking, this should be in a separate commit, as it isn't part of the fix. But it seems too minor to create a separate commit just for that.